src/main/java/entities/Tag.java

```diff
@@ -69,7 +80,16 @@ public void setColor(Color color) {
     /**
      * Gets the owner of the Tag
      */
-//    public User getUser() {
-//        return this.user;
-//    }
+    public User getUser() {
```

Comment:
Ideally this change should have been made by Chhavi, but seeing that it was made in correspondence with her, it seems appropriate.